deploy/crds/apps.openshift.io_servicebindingrequests_crd.yaml

```diff
@@ -211,9 +211,13 @@ spec:
                               for env vars'
                             type: string
                           divisor:
+                            anyOf:
+                            - type: integer
+                            - type: string
                             description: Specifies the output format of the exposed
                               resources, defaults to "1"
-                            type: string
+                            pattern: ^(\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))(([KMGTPE]i)|[numkMGTPE]|([eE](\+|-)?(([0-9]+(\.[0-9]*)?)|(\.[0-9]+))))?$
```

Comment:
regex ?

Reply:
auto-generated by `operator-sdk generate crd`

Reply:
can you check once by removing this? because in a crd we should have the API details and here extra field `anyOf` is being introduced

Reply:
Done 25b27f4